aiserver.py

```diff
@@ -460,40 +460,25 @@ def replace_placeholders(self, ph_ids: dict):
                 self.world_infos[i][key] = self._replace_placeholders(self.world_infos[i][key])
 
     def from_club(self, club_id):
-        # Maybe it is a better to parse the NAI Scenario (if available), it has more data
-        r = requests.get(f"https://aetherroom.club/api/{club_id}")
-
-        if not r.ok:
-            print(f"[import] Got {r.status_code} on request to club :^(")
-            message = f"Club responded with {r.status_code}"
-            if r.status_code == "404":
+        from importers import aetherroom
+        import_data: aetherroom.ImportData
+        try:
+            import_data = aetherroom.import_scenario(club_id)
+        except aetherroom.RequestFailed as err:
+            status = err.status_code
+            print(f"[import] Got {status} on request to club :^(")
+            message = f"Club responded with {status}"
+            if status == "404":
```

Comment:
404 was a string in the old code too, but now that i look at it im pretty sure its an int and this code will never ever run

Reply:
yeah that makes sense. I've fixed the 404 handler to work with ints.